Test on chain sync (#2050)
valory-xyz/open-autonomy: docs/advanced_reference/commands/autonomy_service.md

    - `GOERLI_CHAIN_RPC` : RPC endpoint for the Görli testnet chain.

`--use-custom-chain`

> : Use the custom-chain profile to interact with the {{ autonolas_protocol }} registry contracts. This profile requires that you define some parameters and [contract addresses](../on_chain_addresses.md) as environment variables (see also the {{ autonolas_protocol }} documentation for more information):

Reviewer: ```suggestion
: Use the custom-chain profile to interact with the Autonolas Protocol registry contracts. This profile requires that you define some parameters and [contract addresses](../on_chain_addresses.md) as environment variables (see also the {{ autonolas_protocol }} documentation for more information):
```

    - `CUSTOM_CHAIN_RPC` : RPC endpoint for the custom chain.
    - `CUSTOM_CHAIN_ID` : chain ID.
    - `CUSTOM_COMPONENT_REGISTRY_ADDRESS` : Custom Component Registry contract address.
    - `CUSTOM_AGENT_REGISTRY_ADDRESS` : Custom Agent Registry contract address.
    - `CUSTOM_REGISTRIES_MANAGER_ADDRESS` : Custom Registries Manager contract address.
    - `CUSTOM_SERVICE_MANAGER_ADDRESS` : Custom Service Manager contract address.
    - `CUSTOM_SERVICE_REGISTRY_ADDRESS` : Custom Service Registry contract address.
    - `CUSTOM_GNOSIS_SAFE_PROXY_FACTORY_ADDRESS` : Custom Gnosis Safe multisig contract address.
    - `CUSTOM_GNOSIS_SAFE_SAME_ADDRESS_MULTISIG_ADDRESS` : Custom Gnosis Safe Same Address Multisig address.
    - `CUSTOM_SERVICE_REGISTRY_TOKEN_UTILITY_ADDRESS` : Custom Service Registry Token Utility address.

>     - `CUSTOM_MULTISEND_ADDRESS` : Custom Multisend address.

Reviewer 2: same comment above

!!! note

>     For L2 chains you will only require to set `CUSTOM_SERVICE_REGISTRY_ADDRESS`, `CUSTOM_GNOSIS_SAFE_PROXY_FACTORY_ADDRESS`, `CUSTOM_GNOSIS_SAFE_SAME_ADDRESS_MULTISIG_ADDRESS` and `CUSTOM_MULTISEND_ADDRESS`

Reviewer 2: same comment above

Author: Addressed [c241c59](https://github.com/valory-xyz/open-autonomy/pull/2050/commits/c241c59d71cdbcfc957398e1e6eb69eaef5daab1)

> !!! note
> 
>     For L2 chains you will only require to set `CUSTOM_SERVICE_MANAGER_ADDRESS`, `CUSTOM_SERVICE_REGISTRY_ADDRESS`, `CUSTOM_GNOSIS_SAFE_PROXY_FACTORY_ADDRESS`, `CUSTOM_GNOSIS_SAFE_SAME_ADDRESS_MULTISIG_ADDRESS` and `CUSTOM_MULTISEND_ADDRESS`

Reviewer: ```suggestion
    !!! note

        For L2 chains you are only required to set

        - `CUSTOM_SERVICE_MANAGER_ADDRESS`,
        - `CUSTOM_SERVICE_REGISTRY_ADDRESS`,
        - `CUSTOM_GNOSIS_SAFE_PROXY_FACTORY_ADDRESS`,
        - `CUSTOM_GNOSIS_SAFE_SAME_ADDRESS_MULTISIG_ADDRESS` and
        - `CUSTOM_MULTISEND_ADDRESS`.
```

Same as above, but using a hardware wallet:

```bash
autonomy service activate 42 --hwi
```

If an ERC20 token was used as bonding token you'll have to provide the token address using `--token` flag:

```bash
autonomy service activate 42 --key my_key.txt --token <token_address>
```

Make sure your account holds enough tokens for activating the service:

## `autonomy service register`

Register an agent instance in a service minted and activated in the Autonolas Protocol.

> ### Usage

Reviewer 2: do we need to say somewhere that before using this an approval is required?

Author: Approval will be taken care by the framework, the user just needs to have enough funds

Same as above, but using a hardware wallet:

```bash
autonomy service activate 42 --hwi
```

> If an ERC20 token was used as bonding token you'll have to provide the token address using `--token` flag:

Reviewer: ```suggestion
If an ERC20 token was used as bonding token you have to provide the same token address using `--token` flag:
```

```bash
autonomy service activate 42 --key my_key.txt --token <token_address>
```

> Make sure your account holds enough tokens for activating the service:

Reviewer: ```suggestion
Make sure your account holds enough funds for activating the service:
```

`-a, --agent-id AGENT_ID`
: Canonical agent ID.

`--token ERC20_TOKEN_ADDRESS`

> : ERC20 token to use as bond for the agent instances.

Reviewer: ```suggestion
: ERC20 token to use as bond for the agent instances. You must specify the same token used when minitng the service. See the [`autonomy mint service`](./autonomy_mint.md#autonomy-mint-service) command.
```

When providing the agent instance address make sure that the address you provide is funded.

> If an ERC20 token was used as bonding token you'll have to provide the token address using `--token` flag:

Reviewer: ```suggestion
If an ERC20 token was used as bonding token you have to provide the same token address using the `--token` flag:
```

```bash
autonomy service register -i 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 -a 56 42 --hwi --token <token_address>
```

> Make sure your account holds enough tokens to pay for the agent bonds.

Reviewer: ```suggestion
Make sure your account holds enough funds to pay for the agent bonds.
```
